content/en/events/isc-2025.md

```diff
@@ -5,14 +5,66 @@ date: 2025-11-13
+### Save the Date - Thursday, November 13th
 A Global Celebration – 10 Years of InnerSource Commons
 
-We’re thrilled to invite you to a special milestone: the 10-year anniversary of InnerSource Commons! This year’s summit will be a truly global event, taking place across three locations—Yokohama, Berlin, and New York—on the same day.
+We’re thrilled to invite you to a special milestone: the 10-year anniversary of InnerSource Commons! This year’s summit will be a truly global event, taking place across three locations—**Yokohama, Berlin, and New York**—on the same day.
 Come and attend at any of these three locations or join the online livestream.
 
 To mark this exciting occasion, we’re bringing together our vibrant community on three continents, giving everyone the chance to connect, collaborate, and celebrate a decade of InnerSource together.
 
 Save the date and join us on November 13th in the city nearest you!
 
-More information on speakers, keynotes, and venues will be announced soon.
\ No newline at end of file
+Registration will open shortly. More information on speakers, keynotes, and venues will be announced soon.
+
+### Why Sponsor?
+We sincerely appreciate your interest in sponsoring the InnerSource Commons Summit 2025. This landmark event celebrates our 10th anniversary and will take place simultaneously across three global locations - Yokohama, Berlin, and New York - on Thursday, November 13th, 2025.
+# A Unique Global Experience
+This summit represents a first-of-its-kind opportunity to connect with the InnerSource community across three continents simultaneously, offering unprecedented reach and impact for sponsors.
+# Connect with a Global InnerSource Audience
+Participants will join both in-person across our three city hubs and via livestream, creating a truly international gathering of InnerSource practitioners, leaders, and enthusiasts.
+# Join Leading Technology Organizations
+Each year, top companies and organizations in tech sponsor our events. We invite you to join them in 2025 for exceptional exposure and networking opportunities with the global InnerSource community.
+# World-Class Venues in Tech Innovation Centers
+Our three host cities—Yokohama, Berlin, and New York—represent major global hubs for technology and innovation, placing your brand at the center of the conversation.
+# Fair Pricing with Exceptional ROI Potential
+We prioritize fair pricing structures designed to maximize your return on investment. Our multi-location format offers the unique opportunity to target specific regional markets or achieve global reach in a single event.
+# Celebrate a Decade of InnerSource Innovation
+By sponsoring, you'll be supporting and connecting with the community that's been driving InnerSource adoption and best practices for the past decade.
+For more information about sponsorship opportunities, contact Addie Girouard, Director of Sponsorship at Addie@InnerSourceCommons.org
```

Comment:
```suggestion
For more information about sponsorship opportunities, contact Addie Girouard, Director of Sponsorship at addie@innersourcecommons.org
```

Is it just me?  Or does all-lowercase emails look more professional?

Reply:
I think both are acceptable, but I think the technically correct one is addie@innersourcecommons.org
I leave it to you to decide which is which from an English perspective!